core/src/main/resources/META-INF/resources/primefaces-extensions/suneditor/suneditor-widget.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * PrimeFaces SunEditor component
+ * API reference http://suneditor.com/sample/html/out/document-editor.html
+ *
+ * @author Matthieu Valente
+ */
+PrimeFaces.widget.ExtSunEditor = PrimeFaces.widget.DeferredWidget.extend({
+
+    /**
+     * Initializes the widget.
+     *
+     * @param {object} cfg The widget configuration.
+     */
+    init: function(cfg) {
+        this._super(cfg);
+        this.input = $(this.jqId);
+        this.disabled = (cfg.disabled === undefined) ? false : cfg.disabled;
+
+        if (this.disabled) {
+            this.input.attr("disabled", "disabled");
+            this.jq.addClass("ui-state-disabled");
```

Comment:
Shouldn't this go to the enable / disable functions?

Reply:
OK so TextEditor in PF does this too.  Its so it never bothers Creating the QUillJS object if the component was disabled so it renders the DIV and TextArea Disabled without even calling SUNEDITOR to create the editor.  He followed what PF TextEditor is doing for right or wrong?

Reply:
Alright. So it seems there is a small bug in PF then.